### DISCUSSION TITLE
Git Hub Pages

### DISCUSSION THREAD

**Discussion by TurnipHeadd** [community/community#61769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I uploaded my index.html, my home.css, and my images but, my CSS and images do not appear, https://github.com/TurnipHeadd/JavaBeen.github.io.git I have tried countless things, I can view my CSS and images perfectly fine when testing it on visual code but, it seems to not link them here, the text is all the same in my local folder also.

**Comment by radar-blip** [comment]
You have the link to your `home.css` pointing to a folder, since there is no folder,
this should work:
`<link href="home.css" rel="stylesheet" type="text/css">`

Do the same with your images, like so:
`<img class="InstaLogo" src="insta.png" alt="Instagram Link Image">`
